athletics.lic

```diff
@@ -121,19 +126,21 @@ class Athletics
         end
         return if remaining_attempts.zero?
       end
-      Flags.add('ct-climbing-finished', 'You finish practicing your climbing')
+      Flags.add('ct-climbing-finished', 'You finish practicing your climbing', 'The rope\'s will quickly fades away')
       Flags.add('ct-climbing-combat', 'You are engaged in combat')
-      bput("climb practice #{target}", 'You begin to practice ')
+      fix_standing
+      bput("play #{UserVars.song} on zills", 'You begin', 'You\'re already playing a song') if target.eql?('climbing rope') 
```

Comment:
There's a ton of logic in `crossing-training::play_song?` that we're missing here (i.e. edge cases that are handled). Should that be extracted and reused here?